tools/ci_build/gen_def.py

```diff
@@ -57,6 +57,7 @@ def parse_arguments():
    file.write("#include <onnxruntime_c_api.h>\n")
    for c in args.config:
       file.write("#include <core/providers/%s/%s_provider_factory.h>\n" % (c,c))
+   file.write("#include <adapter/WinMLAdapterFactory.h>\n")
```

Comment:
Is this file packed in any of our release packages? Do we need to do so or plan to do it?

Reply:
It isn't packed in any release packages yet. There will need to be a follow up change to change release packages to include the header